composer.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "simonkub/laravel-sipgate-notifications",
```

Comment:
Know you mentioned this but i've just commented here to make sure we don't forget (packagist gets mega confused).

Reply:
updated package name to `laravel-notification-channels/sipgate`, namespace to `NotificationChannels\Sipgate` and package url to `https://github.com/laravel-notification-channels/sipgate`